src/app/GitUI/RepositoryHistoryUIService.cs

```diff
@@ -5,11 +5,37 @@
 
 namespace GitUI
 {
-    internal class RepositoryHistoryUIService
+    /// <summary>
+    /// Represents a service for managing the git repository history.
+    /// </summary>
+    public interface IRepositoryHistoryUIService
+    {
+        /// <summary>
+        ///  Occurs whenever the git module changes.
```

Comment:
Either one or two spaces - everywhere.

Reply:
As Visual Studio by default adds no space at all and there is no formatter, all of this has to be implemented and enforced manually, there is not even a styleguide. Why add spaces?

Reply:
Yes, VS adds a single space by default. However, it was found by numerous engineers that double space significantly increases readability (especially for large blobs of text).